packaging/linux/docker/standard/entrypoint.sh

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
+set -e
```

Comment:
euo pipefail?

Reply:
k ill do full set and i might revert it if it breaks